src/plugins/discover/public/application/main/hooks/use_inspector.ts

```diff
@@ -37,24 +32,19 @@ export function useInspector({
   const onOpenInspector = useCallback(() => {
     // prevent overlapping
     stateContainer.internalState.transitions.setExpandedDoc(undefined);
+    const inspectorAdapters = stateContainer.dataState.inspectorAdapters;
```

Comment:
Inspector flyout is missing a title

<img width="500" alt="Screenshot 2023-03-27 at 14 37 24" src="https://user-images.githubusercontent.com/1415710/227949568-f654afd7-28f7-444a-adb2-a11bc57e1477.png">

Reply:
Thx for this catch, that was an easy one fix ... seems saved search title should be undefined and not ` ''`

Reply:
Works now, thanks!